Dockerfile

```diff
@@ -13,9 +15,14 @@ RUN ( \
     rm -f /lib/systemd/system/basic.target.wants/*; \
     rm -f /lib/systemd/system/anaconda.target.wants/*; \
     touch /etc/sysconfig/network
-COPY sbin/export_environment.sh /sbin/export_environment
-COPY systemd/* /etc/systemd/system/
-RUN chmod +x /sbin/export_environment
+
+COPY sbin/ /sbin/
+COPY etc/ /etc/
 RUN systemctl enable export-environment.service
-VOLUME ["/sys/fs/cgroup"]
-CMD ["/usr/sbin/init"]
+
+ENV ENV_EXPORT_PATH=/etc/ci-container.environment
+ENV ARGS_EXPORT_PATH=/etc/ci-container.args
+# A list of variables to be exported to $export_path
+ENV ENV_INCLUDE_LIST="ENV_EXPORT_PATH ARGS_EXPORT_PATH"
```

Comment:
This change implies that each image that is going to be based on the systemd image, and want to include it's own env vars, will have to redeclare the `ENV_INCLUDE_LIST` variable. This is risky since the `ENV_EXPORT_PATH ARGS_EXPORT_PATH` can be omitted when this happens.